src/ansiblelint/rules/no_log_password.md

```diff
@@ -0,0 +1,38 @@
+# no-log-password
+
+This rule ensures playbooks do not write passwords to logs.
+Always set the `no_log: True` attribute to protect sensitive data.
+
+## Problematic Code
+
+```yaml
+---
+- name: Example playbook
+  hosts: localhost
+  tasks:
+    - name: Log user passwords
+      ansible.builtin.user:
+        name: johnd
+        comment: John Doe
+        uid: 1040
+        group: admin
+        password: password
+      no_log: False # <- Sets the no_log attribute to false.
```

Comment:
@ssbarnea Again I hope you don't read this while on PTO. But I wasn't able to trigger this rule, even using the tests. Am I missing something here? Thanks.